openapi.yaml

```diff
@@ -794,7 +843,7 @@ components:
     SuccessResponse:
       type: object
       properties:
-        status:
+        tweetID:
```

Comment:
now [this](https://github.com/mo-pm888/twitter/blob/d9111c28b6620b720566d7e10dd522f2da33b229/openapi.yaml#L149) is wrong, and [this](https://github.com/mo-pm888/twitter/blob/d9111c28b6620b720566d7e10dd522f2da33b229/openapi.yaml#L590), and [this](https://github.com/mo-pm888/twitter/blob/d9111c28b6620b720566d7e10dd522f2da33b229/openapi.yaml#L613), and so on

Reply:
also, it's `tweet_id` in the code, not `tweetID`